### ISSUE TITLE
java.lang.ClassNotFoundException: Didn't find class "androidx.compose.foundation.gestures.snapping.SnapPositionInLayout" on HorizontalCalendar

### ISSUE THREAD

**Issue by hscissors** [kizitonwose/Calendar#576]
### Library information:
 - Version: 2.5.4 (Compose)
 - Compose: (BOM) 2024.06.00
 - Material 3: 1.3.0-beta05

### Describe the bug**
Here's the stacktrace:
```
java.lang.NoClassDefFoundError: Failed resolution of: Landroidx/compose/foundation/gestures/snapping/SnapPositionInLayout;
                  at com.kizitonwose.calendar.compose.CalendarDefaults.pagedFlingBehavior(CalendarDefaults.kt:25)
                  at com.kizitonwose.calendar.compose.CalendarDefaults.flingBehavior(CalendarDefaults.kt:36)
                  at com.kizitonwose.calendar.compose.CalendarKt.Calendar(Calendar.kt:182)
                  at com.kizitonwose.calendar.compose.CalendarKt.HorizontalCalendar(Calendar.kt:83)

[...]
                                     	
Caused by: java.lang.ClassNotFoundException: Didn't find class "androidx.compose.foundation.gestures.snapping.SnapPositionInLayout" on path: DexPathList[[zip file "/data/app/~~Rr2cvohaO97NLiSfGTPiag==/com.doximity.amiondroid.v2-VCvB9qwVn15h1SSP8yJJ9g==/base.apk"],nativeLibraryDirectories=[/data/app/~~Rr2cvohaO97NLiSfGTPiag==/com.doximity.amiondroid.v2-VCvB9qwVn15h1SSP8yJJ9g==/lib/x86_64, /data/app/~~Rr2cvohaO97NLiSfGTPiag==/com.doximity.amiondroid.v2-VCvB9qwVn15h1SSP8yJJ9g==/base.apk!/lib/x86_64, /system/lib64, /system_ext/lib64]]
                  at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:259)
                  at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
                  at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
                  ... 46 more
```

**Comment by hscissors** [comment]
Looks like maybe this is fixed in the beta release, but whenever I changed my version to the beta, gradle throws:
```
Failed to resolve: com.kizitonwose.calendar:compose:2.6.0-beta04 
Show in Project Structure dialog
Affected Modules: app, injector, presentation

```

**Comment by hscissors** [comment]
Got the beta version working. Looks like it is fixed in the 2.6.0-beta04

**Comment by kopoh** [comment]
I have the same problem in compose multiplatform, and this solution dosn't work fo me(
